seed/challenges/01-front-end-development-certification/bootstrap.json

```diff
@@ -8,7 +8,7 @@
       "id": "bad87fee1348bd9acde08712",
       "title": "Use Responsive Design with Bootstrap Fluid Containers",
       "description": [
-        "Now let's go back to our Cat Photo App. This time, we'll style it using the popular Bootstrap responsive CSS framework.",
+        "In the HTML5 and CSS section of FreeCodeCamp we built a Cat Photo App. Now let's go back to our Cat Photo App. This time, we'll style it using the popular Bootstrap responsive CSS framework.",
```

Comment:
Repetitive use of `Cat Photo App`, "we built a Cat Photo App. Now let's go back to our Cat Photo App."